openpgp.d.ts

```diff
@@ -28,8 +28,6 @@ export class Key {
   private keyPacket: PublicKeyPacket | SecretKeyPacket;
```

Comment:
We need to declare this as `public` if we want to allow for single key/subkey decryption